App.js

```diff
@@ -1,61 +1,83 @@
 import { NavigationContainer } from '@react-navigation/native';
 import { createNativeStackNavigator } from '@react-navigation/native-stack';
-import { StatusBar } from 'expo-status-bar';
-import { StyleSheet, Text, View } from 'react-native';
-import { Provider } from 'react-redux';
+import { Button } from 'react-native';
+import { Provider, useDispatch, useSelector } from 'react-redux';
+import { logoutUser } from './components/Auth/Services/client';
+import {
+  authenticateUser,
+  setUserAuthenticationToken,
+} from './redux/authSlice';
 import { store } from './redux/store';
```

Comment:
An alternative to using redux for state management would be to use the React Context API. Not saying this is better than what you have, but just throwing the options out there. More about context: https://beta.reactjs.org/learn/passing-data-deeply-with-context

Reply:
We were originally taught Context then they introduced us to reducers in the final module. I'm more familiar with context but wanted to try out new libraries with this project